src/main.rs

```diff
@@ -123,9 +125,19 @@ fn main() {
         PathDisplay::Absolute => root
     };
 
-    match RegexBuilder::new(pattern)
-              .case_insensitive(!config.case_sensitive)
-              .build() {
+    let mut builder = if !matches.is_present("use-glob") {
+        RegexBuilder::new(pattern)
+    } else {
+        match GlobBuilder::new(pattern).literal_separator(true).build() {
+            Ok(glob) => {
+                // TODO: provide an option to remove (?-u) in the upstream?
```

Comment:
is this still relevant?

Reply:
If it is unnecessary to match arbitrary bytes in the path, then no need to remove (?-u).